src/core/interfaces.ts

```diff
@@ -420,7 +420,14 @@ export namespace CoreChartProps {
   export interface LegendOptions extends BaseLegendOptions {
     bottomMaxHeight?: number;
     position?: "bottom" | "side";
+    /**
+     * When {@link position} is set to "bottom", horizontalAlignment sets the legend horizontal alignment.
```

Comment:
nit: I would simplify this by saying something like 'The property only applies when `position="bottom"`'